### PR TITLE
Add Header Component

### PR DESCRIPTION
**Pull request by RajVarsani**

### CODE REVIEW

File: src/components/general/header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        />
+        <div className={styles.logo}>
+          <Image
+            src={COMPONENT_DATA.logo.src}
+            alt={COMPONENT_DATA.logo.alt}
+            layout="intrinsic"
+          />
+        </div>
+      </div>
+      <div className={styles.links + (opened ? " " + styles.visible : "")}>
```
Comment (escapedcat) [inline src/components/general/header/Header.tsx:33]: nit: Use template literals

**Review by escapedcat — COMMENTED** [review]
Any sort of link hover will come later?
Thanks for the vercel environment!

File: src/components/general/header/Header.tsx (hunk 1 above)
Comment (RajVarsani) [reply src/components/general/header/Header.tsx:33]: > Any sort of link hover will come later? Thanks for the Vercel environment!

Not any as of now. imo mostly nav links with hover have some dropdown or some menu kind of interface generally. I have not seen much links with some hovers in this type of design.
Let me know what you think.

Comment (escapedcat) [reply src/components/general/header/Header.tsx]: We can always add it later on. Just wanted to check.

**Review by escapedcat — APPROVED** [review]
Merge it

**Pull request merged**